sdk/build.gradle.kts

```diff
@@ -60,6 +60,7 @@ dependencies {
   api(project(":plugin-overlay"))
   api(project(":plugin-annotation"))
   api(project(":extension-localization"))
+  api(project(":plugin-lifecycle"))
```

Comment:
Do we actually want to bundle the lifecycle plugin with the sdk? since the lifecycle plugin depends on   `implementation(Dependencies.androidxAppCompat)` and `implementation(Dependencies.androidxLifecycle)`, and could introduce some binary size increase.

Reply:
`androidxAppCompat` is the standard these days, 99% of developers will include them in their apps.
That we rely on lifecycle is also an expected outcome of this plugin.

Reply:
Yeah, androidxAppCompat can moved to app module, so only androidxLifecycle is included. Users could exclude this plugin if they are size sensitive.